Update REAMDE file (#48)
collab-uniba/pynblint: README.md

python -m ipykernel install --user --name myenv
```
Now just select this kernel you created when running the NotebookAnalysisResults notebook.
## Run the web API

> Use the terminal or an Anaconda Prompt for the following steps:

Reviewer: ```suggestion
Use the terminal or an Anaconda Prompt to perform the following steps:
```

> 1. Move to the folder containing the main.py fastAPI module

Reviewer: ```suggestion
1. Move to the folder containing the `main.py` module
```

`main.py` is actually a pynblint module, not a FastAPI module. In that script we use FastAPI

> 2. Run it with the following command:

Reviewer: ```suggestion
2. Start the API server with the following command:
```

> ```c

Reviewer: ```suggestion
```bash
```

uvicorn main:app --reload
```
The reload command makes the server restart after code changes. Only do this for development.<br>

> 3. You will get a response with address and port on which the server is running, otherwise you will get some kind of error

Reviewer: ```suggestion
3. If the server starts correctly, the address and port on which it is running will be printed on your terminal.
```
